birdhouse/optional-components/test-cowbird-jupyter-access/docker-compose-extra.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.4"
+services:
+  jupyterhub:
+    environment:
+      TEST_COWBIRD_JUPYTERHUB_USERNAME: ${TEST_COWBIRD_JUPYTERHUB_USERNAME}
+  test-cowbird-jupyter-access:
+    image: ${DOCKER_NOTEBOOK_WORKFLOW_IMAGE}
+    container_name: test-cowbird-jupyter-access
+    environment:
+      MAGPIE_ADMIN_USERNAME: ${MAGPIE_ADMIN_USERNAME}
+      MAGPIE_ADMIN_PASSWORD: ${MAGPIE_ADMIN_PASSWORD}
+      PAVICS_HOST_URL: https://${PAVICS_FQDN_PUBLIC}
+      WPS_OUTPUTS_DIR: ${WPS_OUTPUTS_DIR}
+      WORKSPACE_DIR: ${DATA_PERSIST_SHARED_ROOT}/${USER_WORKSPACES}
+      TEST_COWBIRD_JUPYTERHUB_USERNAME: ${TEST_COWBIRD_JUPYTERHUB_USERNAME}
+      TEST_COWBIRD_JUPYTERHUB_PASSWORD: ${TEST_COWBIRD_JUPYTERHUB_PASSWORD}
+    volumes:
+      - ${COWBIRD_JUPYTER_ACCESS_DIR}/test_cowbird_jupyter.py:/test_cowbird_jupyter.py
+      - ./optional-components/test-geoserver-secured-access/test-data:/geoserver-test-data:ro
+      - ${DATA_PERSIST_ROOT}:/data
```

Comment:
Shouldn't this be `${DATA_PERSIST_SHARED_ROOT}` because giving write access to the entire `${DATA_PERSIST_ROOT}` was way too much and that's the reason for  `${DATA_PERSIST_SHARED_ROOT}` to exist.

Reply:
Good catch. I think it makes sense, since I use this volume for wps outputs data and the user workspaces which should both be found in the `${DATA_PERSIST_SHARED_ROOT}`. Updated